util/libcrypto.num

```diff
@@ -4222,3 +4222,4 @@ CT_POLICY_EVAL_CTX_get_time             4172	1_1_0d	EXIST::FUNCTION:CT
 CT_POLICY_EVAL_CTX_set_time             4173	1_1_0d	EXIST::FUNCTION:CT
 X509_VERIFY_PARAM_set_inh_flags         4174	1_1_0d	EXIST::FUNCTION:
 X509_VERIFY_PARAM_get_inh_flags         4175	1_1_0d	EXIST::FUNCTION:
+EVP_PKEY_get0_poly1305                  4176	1_1_1	EXIST::FUNCTION:POLY1305
```

Comment:
Not just likely.  If you do a rebase on top of a fresher master, you will see for yourself ;-)